platforms/core-configuration/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/support/delegates/ClientModuleDelegate.kt

```diff
@@ -102,6 +101,7 @@ abstract class ClientModuleDelegate : org.gradle.api.artifacts.ClientModule {
     override fun setTransitive(transitive: Boolean): ModuleDependency =
         delegate.setTransitive(transitive)
 
+    @Deprecated("Deprecated in Java", ReplaceWith("Object.equals(Object)"))
```

Comment:
Using the correct names here should allow automatic replacement:
```suggestion
    @Deprecated("Deprecated in Java", ReplaceWith("this.equals(dependency)"))
```